### PR TITLE
Form inputs and API calls to tokenize card or bank details with Stripe

### PR DESCRIPTION
**Pull request by jonahbron**

### CODE REVIEW

**Review by butterflyhug — APPROVED** [review]

File: frontend/content/power/faq.en.md [hunk 1]
```diff
@@ -28,6 +28,10 @@ Yes.
 
 Managers face additional risks getting involved with organizing. If you're a manager interested in joining, we'd love to have a conversation with you. Please fill out the [manager interest form](https://airtable.com/shr9bTB9G10keL5jY) and a union member will reach out to discuss.
 
+### Can I belong to more than one union?
```
Comment (twilde) [inline frontend/content/power/faq.en.md:31]: Was this supposed to be in this commit?

File: frontend/config.toml [hunk 2]
```diff
@@ -14,6 +14,7 @@ disableKinds = ["taxonomy", "term"]
   images = ["img/static/og-banner.png"]
   title = "Alphabet Workers Union"
   member_count = 700
+  stripe_key = 'pk_test_51I9k6TE1rwuQcCei5yDImwRkr5S6qIzakaDope6nrnEQcTn9CjqmosnWnlwDbRUYMudlYNS0OVsNqpgjbs3MwQbp00FO2VIwzx'
```
Comment (twilde) [inline frontend/config.toml:17]: I assume this will eventually live in a secret?

**Comment by justaliu** [comment]
I do not know typescript, so I can't really review for that, but just wanted to note that by tabbing, I can't get to switching between bank account and card, and on the card option, I can't get to the actual card details. Not sure if you meant to save that stuff for once we have a design, though.

File: frontend/content/power/faq.en.md (hunk 1 above)
Comment (jonahbron) [reply frontend/content/power/faq.en.md]: No, not sure how that got there.  Removed.

File: frontend/config.toml (hunk 2 above)
Comment (jonahbron) [reply frontend/config.toml:17]: This is the publishable key and must be accessible to the front-end, it can't be kept in a secret.

**Comment by jonahbron** [comment]
@justaliu I haven't put much work into keyboard navigability yet, but that'll be important to do at some point.

**Review by twilde — APPROVED** [review]

**Pull request merged**